src/compositions/WorkshopScreening/Modal/Modal.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import './Modal.css';
+
+const checkEligibility = (userDate, question2, history) => {
+    const currentDate = new Date();
+    if (question2 === 'Yes') {
```

Comment:
same with this functionality. This is a great example of configuration over composition which is not what we want to do. Same with comment above we will want to refactor this soon.